bip-update-process.md

```diff
@@ -0,0 +1,646 @@
+* An **Informational BIP** describes a bitcoin design issue, provides general guidelines or other information to the
+  bitcoin community.
+* A **Process BIP** describes a process surrounding bitcoin, or proposes a change to (or an event in) a process. Process
+  BIPs are like Specification BIPs, but apply to topics other than the bitcoin protocol and bitcoin implementations.
+  They often require community consensus, and users are typically not free to ignore them. Examples include procedures,
+  guidelines, and changes to the decision-making process.
+
+## Workflow
+
+The BIP process starts with a new idea for bitcoin. Each potential BIP must have an author (or multiple
+authors)—someone who writes the BIP, gathers feedback, shepherds the discussion in the appropriate forums, and
+finally recommends a mature proposal to the community.
+
+![Status Diagram](bip-update-process/status-diagram.png "Status Diagram for the BIP Workflow")
+
+### Ideation
+
+After having an idea, the author should evaluate whether it meets the criteria to become a BIP, as described in
+this BIP. The idea must be of interest to the broader community or relevant to multiple software projects. Small
+improvements and matters concerning only a single project usually do not require standardization and should instead be
+brought up to the relevant project directly.
+
+The author should first research whether an idea has been considered before. Ideas in bitcoin are often rediscovered,
+and prior related discussions may inform the author what issues may arise in its progression. After some investigation,
+the novelty of an idea can be tested by posting about it to the [Bitcoin Development Mailing
+List](https://groups.google.com/g/bitcoindev).
+
+Vetting an idea publicly before investing the time to formally describe the idea is meant to save both the author and
+the broader community time. Not only may someone point out relevant discussion topics that were missed in the author’s
+research, or that an idea is guaranteed to be rejected based on prior discussions, but describing an idea publicly also
+tests whether it is of interest to more people besides the author. After establishing that the idea may be of interest
+to the bitcoin community, the author should work on drafting a BIP.
+
+It is recommended that complicated proposals be split into separate BIPs that each focus on a specific component of the
+overall proposal.
+
+### Progression through BIP Statuses
+
+As a first sketch of the proposal is taking shape, the author should present it to the [Bitcoin Development Mailing
+List](https://groups.google.com/g/bitcoindev). This gives the author a chance to collect initial feedback and address
+fundamental concerns. If the author wishes to work in public on the proposal at this stage, it is recommended that they
+open a pull request against their fork of the BIPs repository instead of the main BIPs repository.
```

Comment:
In the ideation phase, the would-be authors are just describing the idea, i.e. work on a proposal document has not started. In the section about Progression through the BIP Statuses, the authors are sending a first draft of the BIP to the mailing list, i.e. we are now talking about work on an actual proposal.

It makes sense to me for the "bouncing around an idea" and "writing a proposal" to be distinct sections, but perhaps the distinction was not clear enough. Does that make sense to you? Do you have suggestions how to clarify the distinction?

Reply:
Okay, what I found confusing is that below "Progression through BIP Statuses", even though it discusses a phase before any BIP status is assigned (and when the authors still work in their own repo). I think the proper way to resolve this is to move this paragraph to "Ideation". 

Then, if you like to distinguish "bouncing around an idea" and "writing a proposal", you could create two subsections of "Ideation". That's okay, but if you ask me, I think that's a tiny bit overkill because we don't need to formalize the Ideation part too much -- it's precisely part of the "official" process. I assume the reason why you talk about it at length is to help authors follow best practices and tell them to do their research upfront. So I think it's just okay that this phase of writing an initial proposal external to the BIPs repo belongs just to "Ideation", with the idea that "Ideation" is simply everthing before the official process. Perhaps there's a bit better term for "Ideation" that captures this writing step, too, but I can't think of one.

Reply:
Ah gotcha, thanks for elaborating, that makes sense and I followed your advice to move the paragraph to ideation.